src/helper/filterFunctions.js

```diff
@@ -163,12 +168,29 @@ export const checkFilterValues = (filterData, userData) => {
   } else if (filterData !== '' && typeof filterData === 'string') {
     return userData
       ? userData
-        .toLowerCase()
-        .normalize('NFD')
-        .replace(/[\u0300-\u036f]/g, '')
-        .includes(filterData.toLowerCase())
+          .toLowerCase()
+          .normalize('NFD')
+          .replace(/[\u0300-\u036f]/g, '')
+          .includes(filterData.toLowerCase())
       : false;
   } else {
     return true;
   }
 };
+
+export const years = () => {
+  let years = [];
+  const currentYear = moment().year();
+  for (let i = currentYear; i >= 2000; i--) {
+    years.push(i);
+  }
+  return years;
+};
+
+export const months = () => {
```

Comment:
why not:
`export const months () => [1,2,3,4,5,6,7,8,9,10,11,12]`

or even months = [1,2,3,4,5,6,7,8,9,10,11,12], if you rather accept array

Reply:
With function it is easier to manage, e.g. if we want to show month names instead of numbers in the future